plugins/block-test/src/mutators.js

```diff
@@ -50,23 +50,23 @@ const COLOR_CHANGE_MUTATOR = {
    * @this {Blockly.Block}
    */
   domToMutation: function (xmlElement) {
-    this.setColour(xmlElement.getAttribute('colour'));
+    this.setFieldValue('LABEL', xmlElement.getAttribute('arbitrary'));
```

Comment:
I think these params are reversed, should be value then name. https://developers.google.com/blockly/reference/js/blockly.block_class.setfieldvalue_1_method.md

Reply:
Good catch. I went back to the test page and for each mutator test block I dragged into the workspace, opened the mutator, modified the block, closed the mutator, and then copy/pasted the block to confirm it was correct.